gatorgrade/output/output_functions.py

```diff
@@ -1,7 +1,35 @@
-"""Add docstring."""
+"""This module is used for storing the main functions requested from the Github Issue Tracker for the output team.
+For instance, functions dealing with percentage output, description output, and colorization of text."""
 
+import gator
+import os
+import output_tools
+import List
+import Tuple

```

Comment:
this can be rephrased to say:
```suggestion
from typing import List, Tuple
```